config/capi/_ytt_lib/capi-k8s-release/templates/secrets.lib.yml

```diff
@@ -22,7 +25,8 @@
 
 #@ ccng_config_mount_path = "/config/cloud_controller_ng.yml"
 #@ ccng_secrets_mount_path = "/config/secrets.yml"
-#@ ccdb_mount_path = "/etc/cf-api/ccdb"
+#@ ccdb_password_mount_path = "/etc/cf-api/ccdb_password"
+#@ ccdb_encryption_key_mount_path = "/etc/cf-api/ccdb_encryption_key"
```

Comment:
Why are there no extensions on the new files' paths? Is it because they're simple plain text files?

Reply:
@ericpromislow They're directories, with files inside for each different field. In this case the only file is `password`, which we add to the variable later in this same yaml file.

Reply:
right, thx